cpp/ql/src/semmle/code/cpp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -268,6 +268,9 @@ newtype TTranslatedElement =
       ) or
       exists(ThrowExpr throw |
         throw.getExpr().getFullyConverted() = expr
+      ) or
+      exists(LambdaExpression lambda |
+        lambda.getChild(0).getFullyConverted() = expr
```

Comment:
If `getChild(0)` is a useful value to query for a `LambdaExpression`, there should be a getter for it.

Reply:
Added `LambdaExpression.getInitializer()`